CMakeLists.txt

```diff
@@ -290,6 +290,9 @@ CMAKE_DEPENDENT_OPTION(
   "Slicer_BUILD_ITKPython" OFF)
 mark_as_superbuild(Slicer_INSTALL_ITKPython)
 
+option(Slicer_BUILD_PARAMETERSERIALIZER_SUPPORT "Build Slicer with parameter serializer support" OFF)
```

Comment:
> I just had to add #include "vtk_jsoncpp.h"

I do not think this is a good idea. jsoncpp is a much smaller dependency than VTK. There will be packages that are not dependent on Slicer or VTK, but dependent on jsoncpp.

I don't see what are the harms of including external jsoncpp project.